build.gradle

```diff
@@ -2258,6 +2267,10 @@ project(":graphics") {
     project.ext.moduleRuntime = true
     project.ext.moduleName = "javafx.graphics"
 
+    // FIXME: Remove this setting when JDK-8334137 is fixed
```

Comment:
perhaps refer to https://bugs.openjdk.org/browse/JDK-8334161 as well here?

Reply:
Probably not in this case. We generally don't refer to the bug ID that added a line of code or a comment.

I will add a comment to [JDK-8334137](https://bugs.openjdk.org/browse/JDK-8334137) with a note to remove this block as part of fixing it.